The edit button on the list screen has been moved to the detail screen

csKyou opened the pull request:
close #95
close #96
close #97
close #98
close #99 
close #100
close #106

csKyou committed:
I moved the edit button on the box list screen to the detail screen

csKyou committed:
I moved the edit button on the role list screen to the detail screen

csKyou committed:
I moved the edit button on the account list screen to the detail screen

csKyou committed:
I moved the edit button on the relation list screen to the detail screen

csKyou committed:
Deleted edit button from ExternalCell and ExternalRole list screen

csKyou committed:
I added a detailed screen under the Rule list screen.
I moved the edit button on the rule list screen to the detail screen.

csKyou committed:
Fixed a bug that token refresh processing may not work correctly

csKyou committed:
Added an edit button on ExternalCell detail screen.
close #102

csKyou committed:
We re-implemented the process to refresh the token every fixed time
 - Fixed a phenomenon that token may be interrupted when updating the browser

dixonsiu commented on src/js/cellProfile.js:
```diff
@@ -1133,6 +1133,9 @@ $(function() {
+            let ManagerInfo = JSON.parse(sessionStorage.ManagerInfo);
+            ManagerInfo.refreshToken = getClientStore().refreshToken;
+            sessionStorage.ManagerInfo = JSON.stringify(ManagerInfo);
         }, 3480000); //58 Minutes.3480000
```
Change to 30 minutes

dixonsiu commented on src/js/uBox.js:
```diff
@@ -554,7 +543,9 @@ function updateBox() {
 				isBoxNotExist = null;
-				editBox(boxName, body, etag,objBoxManager);
+				if (editBox(boxName, body, etag,objBoxManager)) {
+					updateBoxInfo(newBoxName, newschemaUrl);
```
Check if line 536 also needs to call updateBoxInfo.

csKyou committed:
・The edit button on the list screen has been moved to the detail screen
・Fixed a bug that token refresh processing may not work correctly
 - review Correspondence

dixonsiu commented on src/js/roleToAccountMapping.js:
```diff
@@ -23,6 +23,19 @@ var arrDeletedConflictCount = [];
 var isDeleted = false;
 
+roleToAccountMapping.prototype.setRoleName = function(name) {
+	roleName = name;
```
These global variables (roleName, boxName) are used within this files, too. Should replace them with set/get methods.

csKyou committed:
Added edit button to ExternalRole details screen
close #103

csKyou committed:
Delete unnecessary processing
